tests/network_policy_test.go

```diff
@@ -117,7 +116,6 @@ func TestNetworkPolicyCoder(t *testing.T) {
 			if test.ExpectDatabasePolicy {
 				require.Contains(t, policy.Spec.PolicyTypes, networkingv1.PolicyTypeIngress, "expected to restrict ingress")
 				require.Contains(t, policy.Spec.PolicyTypes, networkingv1.PolicyTypeEgress, "expected to restrict egress")
-				require.Empty(t, policy.Spec.Egress, "expected empty egress rules")
```

Comment:
should add an assertion that the policy is as we expect, the Go struct is probably more obvious about how it's working than the YAML